Ginger/GingerCoreCommon/VariablesLib/VariableBase.cs

```diff
@@ -217,12 +194,12 @@ public string Formula
         public string ParentName { get; set; }
         private bool mVarValChanged;
         [IsSerializedForLocalRepository]
-        public bool VarValChanged { get { return mVarValChanged; } set { mVarValChanged = value; OnPropertyChanged(Fields.VarValChanged); } }
+        public bool VarValChanged { get { return mVarValChanged; } set { mVarValChanged = value; OnPropertyChanged(nameof(VarValChanged)); } }
 
         //used to identify the variables which the user customized for specific BF run
         private bool mDiffrentFromOrigin;
         [IsSerializedForLocalRepository]
-        public bool DiffrentFromOrigin
+        public bool DifferentFromOrigin
```

Comment:
You can't change name of fields which been serilized to XML because then previous saved XML's will have old name of it and field unserilize will fail because it won't found the matching field for it.

i see you doing a lot of typo fixing but please do not do it to fields which has the  [IsSerializedForLocalRepository] attribute

Reply:
only this one, did you changed any other serialized field name?